lib/GADS/Schema/Result/User.pm

```diff
@@ -866,7 +866,7 @@ sub update_user
 
     if(defined $params{account_request}) {
         $values->{account_request} = $params{account_request};
-        $request = 1;
+        $request = 1 if $self->account_request != $params{account_request};
```

Comment:
While I understand that this will get the same result (because we want the one in the `$self` to be 1, with the `$params` one being 0), I wonder what the difference is?

Either way, done! :smile:

Reply:
Purely from the point of view of understanding the code, without understanding any of the underlying logic. If I look at the first version I say "we send welcome email if: either it's not an account request and params is an account request, or it is an account request and the params is not for an account request". For the altered version, I think it's more accurate, as we say "We send welcome email only if this is an account request and the params is for an account request".